src/main/java/frc/robot/robot_manager/RobotCommands.java

```diff
@@ -82,11 +82,13 @@ public Command stopShootingCommand() {
 
   public Command feedingCommand() {
     return Commands.runOnce(robot::prepareFeedRequest, requirements)
+        .andThen(robot::confirmShotRequest)
```

Comment:
the confirm shot part here is unnecessary and won't do what you want. the previous command is a `runOnce`, so that means the confirm shot request is going to be run instantly.